sdk/identity/azure_identity/README.md

```diff
@@ -1,77 +1,425 @@
-# azure_identity
-
-Azure Identity crate for the unofficial Microsoft Azure SDK for Rust. This crate is part of a collection of crates: for more information please refer to <https://github.com/Azure/azure-sdk-for-rust>.
-
-This crate provides several implementations of the [azure_core::auth::TokenCredential](https://docs.rs/azure_core/latest/azure_core/auth/trait.TokenCredential.html) trait.
-It is recommended to start with `azure_identity::create_credential()?`, which will create an instance of `DefaultAzureCredential` by default. If you want to use a specific credential type, the `AZURE_CREDENTIAL_KIND` environment variable may be set to a value from `azure_credential_kinds`, such as `azurecli` or `virtualmachine`.
-
-```rust,no_run
-use azure_core::credentials::TokenCredential;
-use std::sync::Arc;
-
-#[tokio::main]
-async fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let subscription_id =
-        std::env::var("AZURE_SUBSCRIPTION_ID").expect("AZURE_SUBSCRIPTION_ID required");
-
-    let credential: Arc<dyn TokenCredential> = azure_identity::DefaultAzureCredential::new()?;
-
-    // Let's enumerate the Azure storage accounts in the subscription using the REST API directly.
-    // This is just an example. It is easier to use the Azure SDK for Rust crates.
-    let url = url::Url::parse(&format!("https://management.azure.com/subscriptions/{subscription_id}/providers/Microsoft.Storage/storageAccounts?api-version=2019-06-01"))?;
-
-    let access_token = credential
-        .get_token(&["https://management.azure.com/.default"])
-        .await?;
-
-    let response = reqwest::Client::new()
-        .get(url)
-        .header(
-            "Authorization",
-            format!("Bearer {}", access_token.token.secret()),
-        )
-        .send()
-        .await?
-        .text()
-        .await?;
-
-    println!("{response}");
-    Ok(())
-}
+# Azure Identity client library for Rust
+
+The Azure Identity library provides [Microsoft Entra ID](https://learn.microsoft.com/entra/fundamentals/whatis) ([formerly Azure Active Directory](https://learn.microsoft.com/entra/fundamentals/new-name)) token authentication support across the Azure SDK. It provides a set of [`TokenCredential`][token_cred_ref] implementations that can be used to construct Azure SDK clients that support Microsoft Entra token authentication.
+
+[Source code][source] | [Package (crates.io)][package] | [API reference documentation][identity_api_docs] | [Microsoft Entra ID documentation][entraid_doc]
+
+## Getting started
+
+### Install the package
+
+Install the Azure Identity library for Rust with cargo:
+
+```bash
+cargo add azure_identity
 ```
 
-## Design
+### Prerequisites
+
+* An [Azure subscription][azure_sub].
```

Comment:
Tip: if you keep referencing links this way with (mostly) the same name, you can just use `[Azure subscription]` and define the link as:

```markdown
[Azure subscription]: ...
```

Saves you a lot of typing and looking up what you called it.

Reply:
I see you mention `[Azure CLI][azure_cli]` a lot. Imagine just defining that as `[Azure CLI]` and having to type only that over and over again.

Reply:
Updated `[Azure subscription]` & `[Azure CLI]` links and a few more.